Cargo.toml

```diff
@@ -29,10 +30,10 @@ azure-devops = { project = "cobalt-org", pipeline = "liquid-rust" }
 
 [features]
```

Comment:
Also, can you enable all the features for documentation?

See https://github.com/crate-ci/clap-cargo/blob/master/Cargo.toml#L14